source/resolve-conflicts.ts

```diff
@@ -1,8 +1,9 @@
-// eslint-disable-next-line @typescript-eslint/triple-slash-reference -- TODO: Is any other way supported?
-/// <reference types="codemirror" />
+/* eslint-disable unicorn/better-regex -- Go home you're drunk */
 
-// eslint-disable-next-line @typescript-eslint/no-namespace -- TODO: Is any other way supported?
-declare namespace CodeMirror {
+import regexJoin from 'regex-join';
+import type CodeMirror from 'codemirror';
+
+declare module 'codemirror' {
 	interface LineHandle {
```

Comment:
Did I just find a way to avoid triple-slash references or was the type package improved to provide this export? Either way… fixed 2 line exception 🎊